third_party/blink/renderer/core/dom/scripted_idle_task_controller.h

```diff
@@ -100,6 +100,9 @@ class CORE_EXPORT ScriptedIdleTaskController
                      base::TimeTicks deadline,
                      IdleDeadline::CallbackType);
 
+  // [RUN-1485] Allow asserting on idle task count.
+  unsigned ReplayIdleTaskCount() const { return idle_tasks_.size(); }
```

Comment:
If this member isn't replay specific then the accessor should just be IdleTaskCount().